holmes/plugins/toolsets/json_filter_mixin.py

```
@@ -0,0 +1,126 @@
+    @classmethod
+    def extend_parameters(cls, existing: Dict[str, ToolParameter]) -> Dict[str, ToolParameter]:
+        merged = dict(cls.filter_parameters)
+        merged.update(existing)
+        return merged
+
+    def _filter_result_data(self, data: Any, params: Dict) -> Tuple[Any, Optional[str]]:
+        parsed_data = data
+        if isinstance(data, str):
+            try:
+                parsed_data = json.loads(data)
+            except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)